gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -73,6 +73,13 @@
                 <span translate ng-if="!folderEditor.rescanIntervalS.$valid && folderEditor.rescanIntervalS.$dirty">The rescan interval must be a non-negative number of seconds.</span>
               </p>
             </div>
+            <div class="form-group" ng-class="{'has-error': folderEditor.longRescanIntervalS.$invalid && folderEditor.longRescanIntervalS.$dirty}">
```

Comment:
I don't think we should expose this here. I would suggest something like a checkbox "Use Filesystem Notifications" in addition to the existing "Rescan Interval" box. When checked, rescan interval is greyed out. The long rescan interval lives in the advanced/hidden options only.

Challenge: GUI needs to know whether filesystem notifications are supported at all, in order to show the checkbox...